acceptance/its_fixture_test.go

```
@@ -0,0 +1,479 @@
+		server.Close()
+		return nil, fmt.Errorf("encode Rekor public key: %w", err)
+	}
+	tempDir, err := os.MkdirTemp("", "its-acceptance-")
+	if err != nil {
+		server.Close()
+		return nil, fmt.Errorf("create ITS fixture directory: %w", err)
+	}
+	rekorPublicKeyFile := filepath.Join(tempDir, "rekor.pub")
+	if err := os.WriteFile(rekorPublicKeyFile, rekorPublicKey, 0o600); err != nil {
+		server.Close()
+		os.RemoveAll(tempDir)
```

golangci-lint (2.13.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)